benchexec/model.py

```diff
@@ -719,7 +725,14 @@ def after_execution(self, exitcode, forceTimeout=False, termination_reason=None)
             output = []
 
         self.status = self._analyze_result(exitcode, output, isTimeout, termination_reason)
-        self.category = result.get_result_category(self.identifier, self.status, self.properties)
+        if self.multiproperty:
+            for property in self.properties:
+                short_property = property.replace(result._PROP_CALL_SPECIFIC, '')
```

Comment:
If it is correct only for CPAchecker, then let cpachecker wrapper to do this job. For general case I suggest to pass 'unreach-call,error_function_name' as a property.